docs/docusaurus/versioned_docs/version-1.8.0/feg/s1ap_federated_tests.md

```diff
@@ -74,7 +74,7 @@ fab federated_integ_test
 You can access Orc8r adding to your keychain the `admin_operator.pfx` cert
 you will find at `/magma/.cache/test_certs`. Then you can check your
 provisioned gateways using
-[swagger interface](https://127.0.0.1:9443/apidocs/v1/?docExpansion=none)
```

Comment:
I think this URL also appears in the v1.7 version of this page which I don't think you've updated